### PR TITLE
feat: Added celery to handle scraping tasks in background

### LINKED ISSUE #32: 32-celery-tasks
- [ ] Add functionality to handle scraping tasks to work in the background using celery.
- [ ] Add API endpoint to get task status.

### PR DESCRIPTION
**Pull request by karelumair**
- [x] Add functionality to handle scraping tasks to work in the background using celery.
- [X] Add API endpoint to get task status.
closes #32

### CODE REVIEW

File: utils/scraping.py [hunk 1]
```diff
@@ -47,29 +51,30 @@ def scrape_quotes(url) -> list:
...
+            "dob": get_date(dob),
+            "country": country,
+            "description": description,
+        }
 
-    driver.quit()
-    return author_data
+        author.update(**author_data)
+
+    return True
```
Comment (pratikGhodke1) [inline utils/scraping.py:80]: These changes which includes beautiful soup can be added into another commit as it is completely different from celery tasks handling

Comment (karelumair) [reply utils/scraping.py:80]: Ok.

So, should I keep selenium part as it is??

**Review by pratikGhodke1 — APPROVED** [review]
Look Awesome! Just keep in mind to use different commits for different major refactorings/changes.

**Comment by karelumair** [comment]
> Look Awesome! Just keep in mind to use different commits for different major refactorings/changes.
Sure

File: utils/scraping.py (hunk 1 above)
Comment (pratikGhodke1) [reply utils/scraping.py:80]: Update selenium find_elements to use beautifulSoup methods

**Pull request merged**